tools/requirements/tests.txt

```diff
@@ -1,7 +1,5 @@
 --use-feature=2020-resolver
 cryptography==2.8
-csv23
-enum34; python_version < '3.4'
 freezegun
 mock
```

Comment:
I think we can switch to `unittest.mock` here as well?

Reply:
Yes!

Reply:
Unfortunately not, at least not without other changes.

```
tests/conftest.py
554              call.args[0] for call in self._server.mock.call_args_list
```

Here, the `call` object's `.args` property is accessed, but that wasn't added until Python 3.8.

Changing this line to use tuple unpacking would be less explicit, IMO and less readable.

Once Python 3.7 support is dropped, we can also drop `mock`.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args

Reply:
I think making it a [0][0] access, with a comment saying "hey, change this to .args[0] after 3.7 is dropped" should be sufficient I think. :)